backend/src/LibBackend/Account.fs

```diff
@@ -22,24 +22,9 @@ type Account =
     email : string
     name : string }
 
-type UserInfo =
-  { username : UserName.T
-    name : string
-    admin : bool
-    email : string
-    id : UserID }
-
-type UserInfoAndCreatedAt =
-  { username : UserName.T
-    name : string
-    admin : bool
-    email : string
-    id : UserID
-    createdAt : NodaTime.Instant }
-
-let userInfoToPerson (ui : UserInfo) : LibService.Rollbar.Person =
-  Some { id = ui.id; username = Some ui.username }
+type UserInfo = { id : UserID }
 
+type UserInfoAndCreatedAt = { id : UserID; createdAt : NodaTime.Instant }
```

Comment:
Is this still used? Plausibly we should remove it (can be another PR)

Reply:
Removed